model-optimizer/extensions/front/tf/placeholder_ext.py

```diff
@@ -13,9 +15,30 @@ class PlaceholderFrontExtractor(FrontExtractorOp):
 
     @classmethod
     def extract(cls, node):
+        shape = shape_array([])
+        # Extract output shape from `shape` attribute
+        extracted_shape = tf_tensor_shape(node.pb.attr["shape"].shape)
```

Comment:
what if placeholder is scalar? what will be `node.pb.attr["shape"].shape`?

Reply:
I didn't change this part so it will work as now